docs/guides/feature-changelist-v3.md

```diff
@@ -92,8 +92,4 @@ The following logging resources are removed:
 | Diff | Resource | Attribute | Action Required? |
 |:----:|:--------:|:---------:|:----------------:|
 |(deprecated) | edge_equinix, edge_equinix_ha, edge_csp, edge_csp_ha, edge_neo, edge_neo_ha, edge_platform, edge_platform_ha, edge_zededa, edge_zededa_ha | bandwidth | **Yes**; This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release |
-
-
-
-
-
+|(deprecated) | firenet | keep_alive_via_lan_interface_enabled | **Yes**; This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release |
```

Comment:
Can this be under its own table/header?

If this is for 7.1, it should be R3.1.6
If this is for 7.2, it should be R3.2.0

Reply:
This is under ## R3.2.0 (UserConnect-7.2)**

Reply:
Oh my mistake- the way the diff shows was a little confusing. Ok! LGTM